### PR TITLE
Input focus dropdown display

### PR DESCRIPTION
**Pull request by simensma**
- Added Search Trigger on input focus
- Added Image display of selected item to input element

### CODE REVIEW

File: angucomplete-alt.js [hunk 1]
```diff
@@ -444,17 +451,12 @@
         if (status === 0 || status === -1) { return; }
 
         // normalize return obejct from promise
```
Comment (eburdon) [inline angucomplete-alt.js:453]: *object :p

**Comment by eburdon** [comment]
:+1:

**Comment by blx** [comment]
👍

**Comment by nealclark** [comment]
👍

**Pull request merged**